docs/requirements.txt

```diff
@@ -1,4 +1,3 @@
-sphinx<=1.5.6
+sphinx
 sphinxcontrib-napoleon
```

Comment:
@henry0312 
Could you please help me?

Why we need to install `sphinxcontrib-napoleon`?
As I understand, since Sphinx 1.3 it's buitin extension (http://www.sphinx-doc.org/en/stable/ext/napoleon.html).
`sphinxcontrib-napoleon` is needed to support versions <=1.2: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/ , but in this case we should add in `conf.py` not `sphinx.ext.napoleon` extension, but `sphinxcontrib.napoleon`.
So, we should either don't install old version of the extension, or append extension in `conf.py` by condition (`sphinx.__version__`).

Do I miss something?

Reply:
Really?
Probably, I misunderstood.
Please drop `sphinxcontrib-napoleon`if it is not needed.

Reply:
I've dropped `sphinxcontrib-napoleon` and added min required version according to this: https://github.com/sphinx-doc/sphinx/issues/2436#issuecomment-210761382